todo_app/app.py

```diff
@@ -9,7 +9,14 @@
 @app.route('/')
 def index():
     lst_item = get_items()
-    return render_template('index.html', lst_item=lst_item)
+    return render_template('index.html', lst_item = lst_item)
```

Comment:
`lst_item` sounds like a single item to me. Even calling it `items` would be more clearly a list of items